modules/user_accounts/templates/form_edit_user.tpl

```diff
@@ -36,13 +36,15 @@
     {/if}
 
     <div class="panel panel-default">
-      <div class="panel-body">
+  <div class="panel-body">
        <h3>Password Rules</h3>
-       <ul>
-          <li>The password must be at least 8 characters long</li>
-          <li>The password must contain at least 1 letter, 1 number and 1 character from   !@#$%^&amp;*()</li>
-          <li>The password and the user name must not be the same</li>
-          <li>The password and the email address must not be the same</li>
+      <ul>
+        <li>The password must be at least 8 characters long.</li>
+        <li>The password and the user name must not be the same.</li>
```

Comment:
Optional suggestion would be "The password and the username or email address must not be the same." instead of repeating similar again.